napari/tests/test_mouse_bindings.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+from unittest.mock import Mock
```

Comment:
@jni you mentioned using `pytest` mock, but there doesn't seem to be one built in - it looks like a separate package from here - https://github.com/pytest-dev/pytest-mock - but maybe I'm getting this wrong. Otherwise I think leaving the `unittest` one is fine